### ISSUE TITLE
In the PM UI, the dependencies tree in the details pane incorrectly narrates `>=`

### ISSUE THREAD

**Issue by nkolev92** [NuGet/Home#9852]
In 16.7, P6, Dependencies tree view is not narrated correct. 

Ex: >= 4.2.0 (greater is not getting narrated, only equals is). 

This repros in 16.6 as well.

**Comment by donnie-msft** [comment]
**Repro Steps:**

1. Open a solution in VS.
2. Open NuGet Package Manager.
3. Select a package with dependencies which have a TFM with a lower-bound package version requirement.
eg, NewtonSoft.JSON for Net Standard 1.0/1.3
![image](https://user-images.githubusercontent.com/49205731/127070769-5514c1d2-e829-43fb-a080-996c173bfb43.png)

4. Open Narrator and select one of the items containing ">="
5. Observe that Narrator does not read the greater than symbol by default.

**Comment by donnie-msft** [comment]
Verified with a11y team that this is uncommon behavior among other screen readers.
I've filed a bug against Narrator, as it appears there may be a problem with their TTS engine. Will update once I know more.

**Comment by donnie-msft** [comment]
For anyone experiencing this problem, I found Narrator documentation and after setting “hear punctuation” to **Most**, I then hear “greater than” pronounced by Narrator.
https://support.microsoft.com/en-us/windows/chapter-4-reading-text-8054c6cd-dccf-5070-e405-953f036e4a15

I ran this issue by the Narrator team, and they are considering it "By Design".
